### ISSUE TITLE
NSPersonNameComponentsFormatter.GetLocalizedString(creds.FullName, NSPersonNameComponentsFormatterStyle.Default,NSPersonNameComponentsFormatterOptions.Phonetic) returns blank as string

### ISSUE THREAD

**Issue by softsan** [CrossGeeks/SignInWithAppleSample#2]
In my case, i see a value in `creds.FullName`
`{<NSPersonNameComponents: 0x28027f9c0> {givenName = Sudhir, familyName = Thanki, middleName = (null), namePrefix = (null), nameSuffix = (null), nickname = (null) phoneticRepresentation = (null) }}`

 but after applying 
`appleAccount.Name = NSPersonNameComponentsFormatter.GetLocalizedString(creds.FullName, NSPersonNameComponentsFormatterStyle.Default,NSPersonNameComponentsFormatterOptions.Phonetic);`

it returns "". Please see below screenshots:

<img width="1161" alt="Screenshot 2020-04-05 at 1 24 40 AM" src="https://user-images.githubusercontent.com/7391319/78460188-5c33c200-76dc-11ea-823b-56288c7bf9a0.png">
<img width="1165" alt="Screenshot 2020-04-05 at 1 24 08 AM" src="https://user-images.githubusercontent.com/7391319/78460193-63f36680-76dc-11ea-93cd-e8e42ab193af.png">

**Comment by rezamohamed** [comment]
just a thought, are you doing this on the same app? because if it is, you only get the name and email the first and only ever time you sign in to that app with apple. The workaround is in your appleid you disable the app each time to debug, or (longer way) you have to create a new bundle id. Its not the easiest to work with IMO.

EDIT: I am not the creator of the Source Code, just someone trying to use it

**Comment by softsan** [comment]
just a thought, are you doing this on the same app? 
 -- No i integrated it in my app with almost all logic you described in your code. 
The workaround is in your appleid you disable the app each time to debug,
-- Yeah that is what i'm doing every time.

**Comment by softsan** [comment]
I think i found a way to get the value out of creds.FullName.

```
var firstName = creds.FullName?.GivenName ?? "";
var lastName = creds.FullName?.FamilyName ?? "";
```

this can return first name and last name every time (ofc. i need to stop using Apple Id everytime from settings on iPhone device)
However i still have one question that how can we get profile(user) picture in apple sign in?

**Comment by rezamohamed** [comment]
I don't think there is a profile picture for Apple, your app would need a separate upload for that

**Comment by softsan** [comment]
Yeah looks like you're correct as according to this they don't provide any such info.  https://forums.developer.apple.com/thread/121998

Thanks.

**Comment by rezamohamed** [comment]
@softsan do you know what needs to happen where it says 'Normal app workflow' OnStart in the main view model? https://github.com/CrossGeeks/SignInWithAppleSample/blob/master/SignInWithAppleSample/App.xaml.cs 

This is confusing me because my assumption is that OnInitialized will direct my user appropriately,. I am also finding that the same OnStart code needs to be added to OnResume if you want the app to do something if you Revoked access and restart the app.

**Comment by MobilyteApps** [comment]
I am doing same process but when put password for the apple account nothing is happen?
